runtime/cudaq/platform/mqpu/remote-mqpu.yml

```diff
@@ -0,0 +1,56 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+name: remote-mqpu
+description:
+  "The remote-mqpu target is..."
+
+config:
+  gen-target-backend: true
+  platform-library: mqpu
+  platform-qpu: RemoteSimulatorQPU
+  preprocessor-defines: ["-D CUDAQ_REMOTE_SIM"]
+  link-libs: ["-lcudaq-remote-simulator-qpu"]
+  # Ability to inject bash commands to be executed by the nvq++ main driver script
+  bash: |
+    if ${LIBRARY_MODE}; then
+      # In library mode, we enforce that no custom fallback host compiler has been set via `cmake-host-compiler` option.
+      # Note: even if another Clang-based compiler is set, we cannot guarantee that it is compatible with the LLVM version
+      # that the CUDA-Q runtime is compiled against. Hence, we don't support custom host compiler for this platform.
+      if [[ "$CMAKE_FALLBACK_HOST_CXX" != "$CXX" ]]; then 
```

Comment:
As briefly discussed offline, the option to configure a host compiler will be removed with PR 1603. After this change, the bash code here can be removed such that we have this all be configuration flags only.